src/nvim/keycodes.lua

```diff
@@ -0,0 +1,208 @@
+return {
+  --- @type [string, string][] List of [key, name] tuples.
+  --- For keys with multiple names, put the preferred name first.
+  --- For multiple keys with the same name, put the preferred key first.
+  names = {
+    { [[' ']], 'Space' },
```

Comment:
This could be a dict if the consumer uses `vim.spairs()`.

Reply:
Did you not notice the comments above?